src/main/java/com/elmakers/mine/bukkit/magic/Mage.java

```diff
@@ -1382,13 +1383,21 @@ public void onCast(Spell spell, SpellResult result) {
     @Override
     public float getPower() {
         if (offhandCast && offhandWand != null) {
-            float power = Math.min(controller.getMaxPower(), offhandWand.getPower());
+            float power = Math.min(controller.getMaxPower(), offhandWand.getPower() + getMagePowerBonus());
             return power * powerMultiplier;
         }
```

Comment:
Hm- I'm not sure if I care one way or the other, but should the bonus not be affected by powerMultiplier? It's really only used for /cast so if you're setting power bonus in spells, that means they'll behave differently when cast with /cast - would that be weird?